docs/configuration.md

```diff
@@ -14,45 +14,364 @@ Telliot commands and config file options are as the following:
-* `numProcessors` - an integer number of CPU cores/threads to use for mining.
-* `disputeTimeDelta` - how far back to store values for min/max range - default 5 \(in minutes\)
-* `disputeThreshold` - percentage of acceptable range outside min/max for dispute checking - default
-* `psrFolder` - folder location holding your psr.json file, default working directory
-* `ProfitThreshold` - Solutions will only be submitted when the estimated profit margin is greater than the `ProfitThreshold` setting.  The estimated profit margin (%) is defined as `100*(EstimatedRewards - EstimatedTransactionCosts)/EstimatedTransaction Costs`.  Setting a ProfitThreshold of 0 will submit solutions if the transaction is estimated to break even `(EstimatedRewards = EstimatedTransactionCosts)`.   Values are estimates only.  There is no guarantee on the estimated values or making a profit.
 
+#### Config file options:
+```json
+{
+	"ApiFile": "(Required: false)  - Default: configs/api.json",
+	"DBFile": "(Required: false)  - Default: db",
+	"DataServer": {
+		"ListenHost": "localhost",
+		"ListenPort": 5000
+	},
+	"EnvFile": "(Required: false)  - Default: configs/.env",
+	"EthClientTimeout": "(Required: false)  - Default: 3000",
+	"GasMax": "(Required: false)  - Default: 10",
+	"GasMultiplier": "(Required: false)  - Default: 1",
+	"HistoryFile": "(Required: false)  - Default: configs/saved.json",
+	"Logger": "(Required: false)  - Default: map[apiOracle:info dataServer:info db:info ops:info pow::info rest:info rpc:info tracker:info]",
+	"ManualDataFile": "(Required: false)  - Default: configs/manualData.json",
+	"Mine": {
+		"Heartbeat": {
+			"Duration": 15000000000
+		},
+		"ListenHost": "localhost",
+		"ListenPort": 9090,
+		"MinSubmitPeriod": {
+			"Duration": 901000000000
+		},
+		"MiningInterruptCheckInterval": {
+			"Duration": 15000000000
+		},
+		"ProfitThreshold": 0,
+		"RemoteDBHost": "",
+		"RemoteDBPort": 0
+	},
+	"ServerWhitelist": "(Required: false)  - Default: []",
+	"Trackers": {
+		"DisputeThreshold": 0.01,
+		"DisputeTimeDelta": {
+			"Duration": 300000000000
+		},
+		"FetchTimeout": {
+			"Duration": 30000000000
+		},
+		"MinConfidence": 0.2,
+		"Names": {
+			"disputeChecker": false,
+			"gas": true,
+			"indexers": true
+		},
+		"SleepCycle": {
+			"Duration": 30000000000
+		}
+	}
+}
+```
+So the default config is as follows:
+```json
+{
+	"ApiFile": "configs/api.json",
+	"DBFile": "db",
+	"EnvFile": "configs/.env",
+	"EthClientTimeout": 3000,
+	"GasMax": 10,
+	"GasMultiplier": 1,
+	"HistoryFile": "configs/saved.json",
+	"Logger": {
+		"apiOracle": "info",
+		"dataServer": "info",
+		"db": "info",
+		"ops": "info",
+		"pow:": "info",
+		"rest": "info",
+		"rpc": "info",
+		"tracker": "info"
+	},
+	"ManualDataFile": "configs/manualData.json",
+	"ServerWhitelist": null
+}
+```
 ### LogConfig file options
 
 The logging.config file consists of two fields: \* component \* level
```

Comment:
We changed that and the config for each component has a field for the log level.

Reply:
Done!